### PR TITLE
Fix the attackby of the supermatter containment core

### LINKED ISSUE #29203: Tongs with supermatter crystal sliver disappear when trying to put the sliver into the containment bin
[Note]: # (<B>Please enter the `Round ID` from the `Status` panel if you can!</B> If you believe the issue to be caused by a testmerge, please report it in its relative PR thread. State what the issue is from a "whats wrong" prospective. Issue reports should clearly allow maintainers to understand whats wrong and how to test/reproduce if that is not obvious. Avoid ambiguity. Start your issue report below both of these lines (or remove them))

73139 Round ID

I was the CE and a traitor, my objective was to steal "a sliver off a supermatter crystal" so a little bit into the round I had some time and privacy, and so I cut a bit off, that worked fine, picked it up with the tongs and got the box/supermatter bin/whatever it's called in the other hand, and then uesed tongs on the supermatter bin and the tongs disappeared with the sliver and the bin remained empty luckily somebody killed themselves trying to get in to do the same objective so I stole their tongs and tried again but it did the same

Summary for ,objective "Steal a sliver of a supermatter crystal" the part where you put it into the box causes your tongs to disappear and your sliver as well

### PR DESCRIPTION
**Pull request by optimumtact**
It was eating your tongs for no apparent reason and also not passing the
tongs into the load proc instead passing in it's own sliver

Fixes #29203

### CODE REVIEW

File: code/game/objects/items/theft_tools.dm [hunk 1]
```diff
@@ -181,10 +181,8 @@
 
 /obj/item/nuke_core_container/supermatter/attackby(obj/item/weapon/hemostat/supermatter/tongs, mob/user)
 	if(istype(tongs))
```
Comment (Cyberboss) [inline code/game/objects/items/theft_tools.dm]: Ugh, why is the parameter hard typed ahhhh!!

**Review by Cyberboss — APPROVED** [review]

**Comment by Xhuis** [comment]
This closes #29203, right?

**Comment by MoreRobustThanYou** [comment]
@Xhuis probably

File: code/game/objects/items/theft_tools.dm (hunk 1 above)
Comment (optimumtact) [inline code/game/objects/items/theft_tools.dm]: a meme so supreme you let out a scream

**Pull request merged**